plugin/contrib/mesos/pkg/scheduler/plugin.go

```diff
@@ -632,12 +632,18 @@ func (k *deleter) deleteOne(pod *Pod) error {
 }
 
 // Create creates a scheduler plugin and all supporting background functions.
-func (k *KubernetesScheduler) NewPluginConfig(terminate <-chan struct{}, mux *http.ServeMux) *PluginConfig {
+func (k *KubernetesScheduler) NewPluginConfig(terminate <-chan struct{}, mux *http.ServeMux,
+	makePodsWatcher func() *cache.ListWatch) *PluginConfig {
```

Comment:
I had the code you suggest. James recommended this solution because we don't have to make the create... function public. Either way is fine I guess.